managed-ledger/src/main/java/org/apache/bookkeeper/mledger/impl/OpAddEntry.java

```diff
@@ -211,13 +211,8 @@ public void safeRun() {
         if (ml.hasActiveCursors()) {
             // Avoid caching entries if no cursor has been created
             EntryImpl entry = EntryImpl.create(ledger.getId(), entryId, data);
-            // EntryCache.insert: duplicates entry by allocating new entry and data. so, recycle entry after calling
-            // insert
```

Comment:
these comments on lines 214 and 215 are still valid and shouldn't be removed.

Reply:
this should be just about reverting #12258 changes to OpAddEntry.
This command will revert OpAddEntry to the state before commit 58697915 (#12558):
```
git checkout 58697915~ managed-ledger/src/main/java/org/apache/bookkeeper/mledger/impl/OpAddEntry.java
```

Reply:
@leizhiyuan this hasn't been resolved yet.

Reply:
> @leizhiyuan this hasn't been resolved yet.

done